packages/palette/src/elements/ProgressDots/ProgressDots.story.tsx

```diff
@@ -0,0 +1,37 @@
+import { storiesOf } from "@storybook/react"
+import React, { useEffect } from "react"
+import { useCursor } from "use-cursor"
+import { ProgressDots } from "./ProgressDots"
+
+const Demo = () => {
+  const { index, handleNext } = useCursor({ max: 5 })
+
+  useEffect(() => {
+    const interval = setInterval(handleNext, 1000)
+    return () => {
+      clearInterval(interval)
+    }
+  }, [])
```

Comment:
Shouldn't `handleNext` be a dependency here?

Reply:
We really need to switch off TSLint in this project and add in the hooks linter.